pkg/mappings.bzl

```diff
@@ -537,6 +537,30 @@ pkg_mklink = rule(
     provides = [PackageSymlinkInfo],
 )
 
+def pkg_mklink(name, target, dest, attributes=None, src=None, **kwargs):
```

Comment:
So, I wonder how necessary this is.  It may be cleaner to provide a migration path than to introduce new logic with an easily missed deprecation marker.

Something like this might work for a majority of cases:

```sh
buildozer -f - <<EOF
rename src target //:%pkg_mklink
EOF
```

Reply:
I like to leave backwards compatible behavior around for a release or two. It makes day of upgrade less painful.

But this is why we need deprecation warnings in Bazel. The default can be to suppress them, but you have run with a flag to make them errors. Then you can spot them and migrate.  Right now, our philosophy or no warnings forces people to look to other sources to know they need to change something.